lib/dor/models/preservable.rb

```diff
@@ -11,11 +11,9 @@ module Preservable
 
     def build_provenanceMetadata_datastream(workflow_id, event_text)
       workflow_provenance = create_workflow_provenance(workflow_id, event_text)
-      dsname = 'provenanceMetadata'
-      ds = datastreams[dsname]
-      ds.label = 'Provenance Metadata' unless datastreams.keys.include?(dsname)
+      ds = datastreams['provenanceMetadata']
+      ds.label ||= 'Provenance Metadata'
       ds.ng_xml = workflow_provenance
-      ds.content = ds.ng_xml.to_s
```

Comment:
i actually think this is still necessary for the content in the datastream to get updated properly.  there are lots of examples where we still do this, and IIRC, it's something that @atz went over pretty thoroughly and attempted to make consistent/correct in the dor-services 4 to 5 upgrade.

i'm also just looking at this chunk of code here, so maybe i don't have the full context and the content gets set elsewhere.  or maybe i'm misremembering and we actually need to get rid of this usage.  but there are lots of example that turn up if you do `git grep "content.*=.*to_s"`.

@atz?